### PR TITLE
cmd/action: atlas schema diff command implementation

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: cmd/action/diff.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package action
```
Comment (a8m) [inline cmd/action/diff.go:1]: License

File: cmd/action/diff.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+import (
+	"context"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	// DiffFlags are the flags used in Apply command.
```
Comment (a8m) [inline cmd/action/diff.go:11]:
```suggestion
	// DiffFlags are the flags used in Diff command.
```

File: cmd/action/diff.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+		FromDSN string
+		ToDSN   string
+	}
+	// DiffCmd represents the diff command.
+	DiffCmd = &cobra.Command{
+		Use:   "diff",
+		Short: "Calculate and print the diff between two schemas.",
+		Long: "`atlas schema diff`" + ` connects to two given databases, inspects
+them, calculates the difference in their schemas, and prints a plan of
+SQL queries to bring the "from" database to the schema of the "to" database.`,
```
Comment (a8m) [inline cmd/action/diff.go:22]: 👌🏻

**Review by a8m — APPROVED** [review]
Beautiful 💯

**Review by yonidavidson — APPROVED** [review]
Cool 💯

**Pull request merged**